src/sidebar/Sidebar.jsx

```diff
@@ -24,6 +24,7 @@ const Sidebar = (props) => {
           </button>
           <div className="sidebar-header">
             <img
+              alt="user-image"
```

Comment:
@arkumari2000, try removing 'image' keyword from 'user-image' property,

Reply:
You want to say alt should be only `"user"` and not `"user-image"` ?

Reply:
@arkumari2000, you can make it 'user_logo'

Reply:
Okay